### ISSUE TITLE
Cannot compile F-Droid build flavor: google-services.json is missing

### ISSUE THREAD

**Issue by S-G-iscoding** [vector-im/riot-android#3287]
Hello,
currently I try to compile your app. I have
- cloned the source code (git clone https://github.com/vector-im/riot-android)
- opened it into Android Studio
- run sdkmanager --licenses 
- clicked on Build/Make Project in Android Studio
While building, the following error message appears:
File google-services.json is missing. The Google Services Plugin cannot function without it. 

 Searched Location: 
~/riot-android/vector/src/appfdroid/debug/google-services.json
~/riot-android/vector/src/debug/appfdroid/google-services.json
~/riot-android/vector/src/appfdroid/google-services.json
~/riot-android/vector/src/debug/google-services.json
~/riot-android/vector/src/appfdroidDebug/google-services.json
~/riot-android/vector/google-services.json
(Original folder (absolute path) is replaced with ~)

This issue appears by using the developing and by using the master branch.
I'm using the Android Studio build from Flathub and the latest Android API.
Can anybody help me with it?

**Comment by S-G-iscoding** [comment]
The "Execution failed for task ':vector:processAppfdroidReleaseGoogleServices'.".
AFAIK the F-Droid flavor doesn't include Google Services.

**Comment by spbdimka** [comment]
@S-G-iscoding did you found solution?

**Comment by S-G-iscoding** [comment]
No, I don't found a solution, because I just have waited for an answer. But current I don't have the necessity of compiling (and changing) the app and even haven't tried it for a long time.
